features/data_converter/empty/feature.ts

```diff
@@ -0,0 +1,44 @@
+import { JSONToPayload } from '@temporalio/common/lib/proto-utils';
+import { Feature } from '@temporalio/harness';
+import * as assert from 'assert';
+import { ApplicationFailure } from '@temporalio/common';
+import expectedPayload from './payload.json';
+import * as wf from '@temporalio/workflow';
+
+const activitiesImpl = {
+  async activity(input: any): Promise<void> {
```

Comment:
nit: This could be an optional parameter or union with undefined or just `undefined`.

```suggestion
  async activity(input?: string): Promise<void> {
```